src/commands/git_secret_hide.sh

```diff
@@ -162,7 +162,7 @@ function hide {
       # encrypt file only if required
       if [[ "$update_only_modified" -eq 0 ]] || [[ "$fsdb_file_hash" != "$file_hash" ]]; then
 
-        local args=( --homedir "$secrets_dir_keys" "--no-permission-warning" --use-agent --yes "--trust-model=always" --encrypt )
+        local args=( --homedir "$secrets_dir_keys" "--no-permission-warning" --use-agent --yes "--trust-model=always" --encrypt --armor )
```

Comment:
What potential side effects on existing users / files it might have? 🤔

Reply:
as long as the armored files are fully interoperable between versions of gnupg, then the differences are
* the secret files are text, not binary
* as result they are larger in file size
* maybe git will show diffs in the armored txt files if you do something like 'git log -p' on them, instead of just saying something like 'binary files differ'

Note also that this feature was requested by a user.

Reply:
I think that this might be a good candidate for configuration 🤔

Reply:
@sobolevn Agree on this being a configuration option. For me, binary by default is more sensible. There's no reason I would want these ASCII armored. Benefits of binary: small, diff doesn't try to show changes, and why not? Git supports binary well.